.evergreen/run-mongodb-aws-test.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+# Don't trace since the URI contains a password that shouldn't show up in the logs
+set -o errexit  # Exit the script with error if any of the commands fail
+
+# Supported/used environment variables:
+#       MONGODB_URI             Set the URI, including an optional username/password to use to connect to the server via MONGODB-AWS
+#                               authentication mechanism
+#       JDK                     Set the version of java to be used.  Java versions can be set from the java toolchain /opt/java
+#                               "jdk5", "jdk6", "jdk7", "jdk8", "jdk9", "jdk11"
+
+JDK=${JDK:-jdk11}
+MONGODB_URI=${MONGODB_URI:-"mongodb://localhost"}
+
+############################################
+#            Main Program                  #
+############################################
+
+echo "Running MONGODB-AWS authentication tests"
+
+export JAVA_HOME="/opt/java/${JDK}"
+
+MONGODB_URI="${MONGODB_URI}/aws?authMechanism=MONGODB-AWS"
```

Comment:
The database name is also unnecessary.  Change to:

   ${MONGODB_URI}/?authMechanism=MONGODB-AWS